.github/workflows/rust.yml

```diff
@@ -7,7 +7,7 @@ env:
   # Minimum supported Rust version (MSRV)
   ACTION_MSRV_TOOLCHAIN: '1.63.0'
   # Pinned toolchain for linting
-  ACTION_LINTS_TOOLCHAIN: '1.63.0'
+  ACTION_LINTS_TOOLCHAIN: '1.65.0'
```

Comment:
This was because clippy 1.63 and 1.64 were ignorning an `#[allow(clippy::useless_conversion)]`. I mentioned this in the new commit message when I force-pushed with this change, but didn't update the PR description.

In any case, I have reverted this change, as using `#[allow(clippy::useless_conversion)]` is no longer necessary.